src/main/java/net/sf/jabref/gui/actions/Actions.java

```diff
@@ -11,7 +11,6 @@
     public static final String ADD_TO_GROUP = "addToGroup";
     public static final String AUTO_SET_FILE = "autoSetFile";
     public static final String BACK = "back";
-    public static final String CHECK_INTEGRITY = "checkIntegrity";
```

Comment:
In GeneralAction, the idea is to run a command based on this (which you probably are aware of). However, the earlier implementation didn't use that as actionPerformed was overridden, so it didn't have any effect at all.